using-python-for-science/intro-to-environments.md

```diff
@@ -307,7 +324,7 @@ So, an easy way to proceed is to start again!
 ### Exercise: nuke, rinse, repeat
 
 Delete the `borked` environment, and create a new one with current versions of
-NumPy, scikit-learn, and scikit-image.
```

Comment:
Why has this line changed? I liked sklearn!!!

Reply:
Because sklearn is annoying and depends only on an ancient version of NumPy (1.7) so it did not, in fact, bork the environment! 😂